inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/support/AnnotationQueueOverviewDataProvider.java

```diff
@@ -113,27 +138,23 @@ else if (getSort().getProperty().equals(headers.get(3))) {
             else if (getSort().getProperty().equals(headers.get(4))) {
                 if (o1.getUpdated() == null) {
                     return dir;
-                }
-                else if (o2.getUpdated() == null) {
+                } else if (o2.getUpdated() == null) {
                     return dir * -1;
-                }
-                else {
+                } else {
                     return dir * (o1.getUpdated().compareTo(o2.getUpdated()));
                 }
-            }
-            else {
+            } else {
                 return 0;
             }
         });
 
 
         //Reset
-        this.shownDocuments.clear();
+        this.shownDocuments = new ArrayList<>();
         shownDocuments.addAll(newList);
 
 
-        if ((int)aFirst + (int)aCount > newList.size())
-        {
+        if ((int)aFirst + (int)aCount > newList.size()) {
```

Comment:
Why cast and not work with ints from the start? Having long in the method signature and then secretly reducing it to int is rather misleading to the user of the method.

Reply:
Required as the method iterator is @Override and needs to be overridden

Reply:
Ok, I see - well, that's annoying :(